homeassistant/components/niu/config_flow.py

```diff
@@ -0,0 +1,87 @@
+"""Config flow for Niu integration."""
+import logging
+
+from niu import NiuAPIException, NiuCloud, NiuNetException, NiuServerException
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+# Define data schema to be used and displayed by the config screen
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {vol.Required(CONF_USERNAME): str, vol.Required(CONF_PASSWORD): str}
+)
+
+
+async def validate_input(hass: core.HomeAssistant, data):
```

Comment:
Please don't partially type a method signature, instead it is preferred to fully type it.